docs-v2/pages/projects/git.mdx

```diff
@@ -216,6 +216,71 @@ To resolve this error:
 No, at this time it's not possible because of how resources are connected during the bootstrapping process from the workflow YAML specification.
 However, this is on our roadmap, [please subscribe to this issue](https://github.com/PipedreamHQ/pipedream/issues/9255) for the latest details.
 
+### Can I migrate my commit history to another GitHub Repository while retaining my workflows in my Pipedream Project?
+
+Yes, this guide shows how to migrate a GitHub-synced project repository in Pipedream to a new repository, preserving the entire commit history in the `production` branch while ensuring that workflows remain unaffected.
+
+#### Assumptions
+- **Current GitHub Repository**: `previous_github_repo`
+- **New GitHub Repository**: `new_github_repo`
+- **Pipedream Project**: `pipedream_project`
+- Basic familiarity with git and GitHub
+- Access to a local terminal (e.g., Bash, Zsh, PowerShell)
+- Necessary permissions to modify both the Pipedream project and associated GitHub repositories
+
+#### Steps
+
+1. **Access Project Settings in Pipedream:**
+   - Navigate to your `pipedream_project`.
+   - Use the dropdown menu on the "Edit" button in the top right corner to access `previous_github_repo` in GitHub.
+2. **Clone the Current Repository Locally:**
+   ```
+   git clone previous_github_repo_clone_url
+   ```
+3. Reset GitHub Sync in Pipedream:
+   - In Pipedream, go to your project settings.
+   - Click on "Reset GitHub Connection".
+4. Set Up New Repository Connection:
+   - Configure a new GitHub Sync to `new_github_repo`.
+5. Clone the New Repository Locally:
```

Comment:
Only headings need capitalization on each word, you can remove the captialization for words:

```suggestion
5. Clone the new repository locally:
```